scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -26,22 +26,25 @@ function sources_lr-pcsx-rearmed() {
 }
 
 function build_lr-pcsx-rearmed() {
-    if isPlatform "neon"; then
-        ./configure --platform=libretro --enable-neon
+    local platform
+    if isPlatform "rpi2" || isPlatform "rpi3"; then
+        platform="$__platform"
     else
-        ./configure --platform=libretro --disable-neon
+        isPlatform "arm" && platform+="armv"
+        isPlatform "neon" && platform+="neon"
     fi
-    make clean
-    make
-    md_ret_require="$md_build/libretro.so"
+    [[ -z "$platform" ]] && platform="unix"
```

Comment:
This shouldn't be required btw

Reply:
You are right, the Makefile takes care of defaulting to `unix` if platform is empty. I added that thinking on not passing an empty `platform=` parameter, but it doesn't really matter as you say.